its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -1504,6 +1444,15 @@ private static void analyzeMavenProject(String projectDirName, String projectKey
     analyzeMavenProject(ORCHESTRATOR, projectDirName, Map.of("sonar.projectKey", projectKey));
   }
 
+  private static void bindProject(String projectName, String projectKey, boolean bindingSuggestionDisabled) {
+    backend.getConfigurationService().didAddConfigurationScopes(new DidAddConfigurationScopesParams(
+      List.of(new ConfigurationScopeDto(CONFIG_SCOPE_ID, null, true, projectName, new BindingConfigurationDto(CONNECTION_ID, projectKey,
+        bindingSuggestionDisabled)))));
+    await().atMost(20, SECONDS).untilAsserted(() -> assertThat(didSynchronizeConfigurationScopes).contains(CONFIG_SCOPE_ID));
+    await().untilAsserted(() -> assertThat(clientLogs.stream().anyMatch(s -> s.getMessage().equals("Stored project analyzer configuration"))).isTrue());
```

Comment:
Please add a big TODO or FIXME mentioning that this is a temporary workaround until SLCORE-396 is fixed